config/config/sc-config.yaml

```diff
@@ -288,6 +288,15 @@ elasticsearch:
   # Create default index templates - kubernetes, kubeaudit, and other
   defaultTemplates: true
   additionalTemplates: {}
+  exporter:
+    resources: {}
+    #   requests:
+    #     cpu: 100m
+    #     memory: 128Mi
+    #   limits:
+    #     cpu: 100m
+    #     memory: 128Mi
+    tolerations: {}
```

Comment:
I wasn't sure what to do with that because [the default values for the chart set it to {}](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/values.yaml#L39). In the end I decided to go with the same default values as they used.

Reply:
Sure, in this case it is fine as there is no validation but when there is validation you will get a similar error saying:
```
 Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Prometheus.spec.tolerations): invalid type for com.coreos.monitoring.v1.Prometheus.spec.tolerations: got "map", expected "array"
```

Reply:
https://github.com/prometheus-community/helm-charts/pull/476

Reply:
I see! Sure, I'll change it then.